week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -121,8 +129,16 @@ function getLocationName() { }
 */
 function journeyPlanner(locations, transportMode) {
   // Implement the function body
+  let newArr = []
+  for (i = 0; i < locations.length; i++){
+     if (isAccessibleByTransportMode(locations[i],transportMode)) {
+       newArr.push(getLocationName(locations[i]));
+   }
+  }
```

Comment:
Good to see that you reused the above functions. Only one tiny mistake which in this case doesn't cause any trouble.
The first parameter of the `isAccessibleByTransportMode` function should only contain the transport modes.
```
  - Accepts two parameters:
     1) First parameter is an array of transport modes
        e.g: ["tube", "river boat"]
```
So before calling that, you should call `getTransportModes(locations[i])` to obtain the transport modes.

But as I said it works without that because none of location names matches with any transport name :)